src/xdocs/config_regexp.xml

```diff
@@ -145,43 +145,43 @@
         <source>
           // This code is copyrighted
         </source>
-        <p> The check. </p>
+          <p> The check. </p>
```

Comment:
http://checkstyle.sourceforge.net/config_regexp.html#Regexp

it was ok, do not change location of P tag.

Reply:
@varunu28 This is still not done.

Reply:
@rnveach I have unindented the p tags. Did I miss something?